app/views/content/non-uk-teachers/ukraine.md

```diff
@@ -47,7 +47,7 @@ If you're coming to the UK from Ukraine, [check what you need to do before you t
 
 ### Get an international relocation payment (IRP) worth £10,000
 
-To be eligible for the IRP, you must be coming to England to teach or train to teach languages or physics in:
+To be eligible for the IRP, you must be coming to England to teach languages or physics in:
 
 * the 2023 to 2024 academic year (which runs from 1 September 2023 to 31 August 2024)
 
```

Comment:
@laurahermionetennant92 There's also a link to the IRP page in the line of text after this so not sure if you've deleted it?

Reply:
Sorry don't understand – IRP still relevant for Ukrainian teachers